src/datasources/network/axios.network.service.spec.ts

```diff
@@ -158,4 +159,61 @@ describe('AxiosNetworkService', () => {
       expect(axiosMock.post).toBeCalledWith(url, data, undefined);
     });
   });
+
+  describe('DELETE requests', () => {
+    it(`delete calls axios delete`, async () => {
+      const url = faker.internet.url();
+      const data = { some_data: 'some_data' };
+
+      await target.delete(url, data);
+
+      expect(axiosMock.delete).toBeCalledTimes(1);
+      expect(axiosMock.delete).toBeCalledWith(url, { data: data });
+    });
+
+    it(`delete forwards unknown error as NetworkOtherError`, async () => {
+      const url = faker.internet.url();
+      const data = { some_data: 'some_data' };
+      (axiosMock.delete as any).mockRejectedValueOnce(new Error('Axios error'));
+
+      await expect(target.delete(url, data)).rejects.toThrowError(
+        new NetworkOtherError('Axios error'),
+      );
+
+      expect(axiosMock.delete).toBeCalledTimes(1);
+      expect(axiosMock.delete).toBeCalledWith(url, { data: data });
+    });
+
+    it(`delete forwards response error as NetworkResponseError`, async () => {
+      const url = faker.internet.url();
+      const data = { some_data: 'some_data' };
+      const error = {
+        response: { data: 'data', status: 100 },
+      };
+      (axiosMock.delete as any).mockRejectedValueOnce(error);
```

Comment:
The cast shouldn't be required

Reply:
Yes! done [here](https://github.com/5afe/safe-client-gateway-nest/pull/146/commits/0b1bb4165478747d8ffa85b5f9c57b07d9c6a657)